bee-common/bee-common/src/packable.rs

```diff
@@ -50,6 +50,29 @@ impl Packable for bool {
     }
 }
 
+impl<T: Packable> Packable for Vec<T>
```

Comment:
Would you mind changing the type parameter to `P` like the Option implementation ?